spec/services/catalog/order_state_transition_spec.rb

```diff
@@ -0,0 +1,81 @@
+describe Catalog::OrderStateTransition do
+  let(:tenant) { create(:tenant) }
+  let!(:order) { create(:order, :tenant_id => tenant.id) }
+
+  let!(:order_item) { create(:order_item, :order_id => order.id, :portfolio_item_id => "1", :tenant_id => tenant.id) }
+  let!(:order_item2) { create(:order_item, :order_id => order.id, :portfolio_item_id => "1", :tenant_id => tenant.id) }
```

Comment:
Technically, I don't think you need the `let!` on all of these, because the `before` is going to call `order_item` which then calls `order` when it's setting its `order_id`. The tests that care about `order_item2` already reference it before the spec runs, so I think it should be safe to just turn these into regular `let`s.

Reply:
Updated accordingly.